.github/workflows/build-publish-develop-pr.yml

```diff
@@ -35,9 +35,25 @@ jobs:
       image-tag: ${{ steps.get-image-tag.outputs.image-tag }}
       release-type: ${{ steps.get-image-tag.outputs.release-type }}
     steps:
-      - name: Checkout repository
+      - name: Setup Github Token
+        id: token
+        uses: smartcontractkit/.github/actions/setup-github-token@ef78fa97bf3c77de6563db1175422703e9e6674f # setup-github-token@0.2.1
+        with:
+          aws-role-arn: ${{ secrets.AWS_OIDC_GLOBAL_READ_ONLY_TOKEN_ISSUER_ROLE_ARN }}
+          aws-lambda-url: ${{ secrets.AWS_INFRA_RELENG_TOKEN_ISSUER_LAMBDA_URL }}
+          aws-region: ${{ secrets.AWS_REGION }}
+          set-git-config: "true"
+
+      - name: Checkout capabilities repository
+        uses: actions/checkout@v4.2.1
+        with: // Pass in token because it's a private repo
+          repository: smartcontractkit/capabilities
+          token: ${{ steps.token.outputs.access-token }}
```

Comment:
```suggestion
        with:
          repository: smartcontractkit/capabilities
          # Pass in token because it's a separate and private repo
          token: ${{ steps.token.outputs.access-token }}
```